recipes/serializers.py

```diff
@@ -0,0 +1,45 @@
+from .models import Recipe, Ingredient
+from rest_framework import serializers
+
+
+class ChoiceField(serializers.ChoiceField):
+    def to_representation(self, obj):
+        return self._choices[obj]
+
+
+class IngredientSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Ingredient
+        fields = ('name',)
+        extra_kwargs = {
+            'name': {'validators': []},
+        }
+
+
+class RecipeSerializer(serializers.ModelSerializer):
+    ingredients = IngredientSerializer(many=True)
+    difficulty = ChoiceField(choices=Recipe.DIFFICULTY_CHOICES)
```

Comment:
You don't need a custom field for this. You can use serializers.CharField(choices=...)